lib/services/ios-debug-service.ts

```diff
@@ -227,12 +236,9 @@ class IOSDebugService implements IDebugService {
         }
     }
 
-    private openDebuggingClient(): IFuture<void> {
+    private openDebuggingClient(fileDescriptor: string): IFuture<void> {
         return (() => {
-            this.$projectDataService.initialize(this.$projectData.projectDir);
-            let platformData = this.$platformsData.getPlatformData(this.platform);
-            let frameworkVersion = this.$projectDataService.getValue(platformData.frameworkPackageName).wait().version;
-
+            let frameworkVersion = this.getProjectFrameworkVersion().wait();            
             let inspectorPath = this.getInspectorPath(frameworkVersion).wait();
```

Comment:
The ```inspectorPath``` is used on the next line where we create the location of the Main.html file
``` let inspectorSourceLocation = path.join(inspectorPath, "Safari/Main.html");```
If you prefer I could rename it to ```inspectorDirectoryPath``` or something similar so that it could be more descriptive.